tools/environment/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/descope/go-sdk/descope"
+	"github.com/descope/go-sdk/descope/client"
+	"github.com/spf13/cobra"
+)
+
+// Descope SDK
+
+var descopeClient *client.DescopeClient
+
+func prepare(projectID string) (err error) {
+	descopeConfig := &client.Config{
+		ProjectID:      projectID,
+		ManagementKey:  os.Getenv(descope.EnvironmentVariableManagementKey),
+		DescopeBaseURL: os.Getenv(descope.EnvironmentVariableBaseURL),
+	}
+	if descopeConfig.ManagementKey == "" {
+		// generate a management key in the Company section of the admin console: https://app.descope.com/settings/company
+		return errors.New("the DESCOPE_MANAGEMENT_KEY environment variable must be set")
+	}
+	descopeClient, err = client.NewWithConfig(descopeConfig)
+	return err
+}
+
+// Command line flags
+
+var Flags struct {
```

Comment:
This is more of a code-style discussion really, but I actually think that for a relatively small tool with all the code residing in the same package, using Uppercase symbols can make the code more readable as it helps differentiate between symbols in the local-scope vs in the package-scope.

Reply:
by local-scope you mean the same file?

in go, there is no difference between package-scope and file-scope, so I would strive for [minimal api as possible](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_strive_for_a_minimal_api_surface)

I think that go reader will read this as an external symbol rather then something that is needed outside the package (because the language spec), so i would discourage to expose those state outside the package use

Reply:
In this case by local scope I mean the scope of a function or block, as opposed to the scope of the entire package. Let's discuss today in person